lesson4/recursion1.js

```diff
@@ -0,0 +1,11 @@
+//sayHello(5) -> 'hello' 'hello' ...x5
+
+const sayHello = (a,i=0)=>{
+  if(i==a){
+    return '';
```

Comment:
this function is just printing.... it should not be returning.

Reply:
done, thanks!